openfisca_us/variables/hhs/ccdf.py

```diff
@@ -67,18 +67,32 @@ class is_enrolled_in_ccdf(Variable):
 
 class is_ccdf_initial_income_eligible(Variable):
     value_type = bool
-    entity = Person
+    entity = SPMUnit
     definition_period = YEAR
     label = u"Initial income eligibility for CCDF"
 
 
 class is_ccdf_continuous_income_eligible(Variable):
     value_type = bool
-    entity = Person
+    entity = SPMUnit
     definition_period = YEAR
     label = u"Continuous income eligibility for CCDF"
 
 
+class is_ccdf_smi_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    definition_period = YEAR
+    label = u"SMI eligibility for CCDF"
+
+
+class is_ccdf_reason_for_care_eligible(Variable):
```

Comment:
what does this mean?

Reply:
Sorry forgot to mention that. It's a requirement that the kid is living with parents who are working or in job training or in educational program, or the kid is receiving or needs protective services. I used 'reason for care' because New York plan uses that term.

![image](https://user-images.githubusercontent.com/26745287/144063119-bd619ec1-dc6d-44b7-b211-1b91ee88c5a8.png)